src/style.css

```diff
@@ -0,0 +1,20 @@
+* {
+  margin: 0;
```

Comment:
reset margins where it is necessary, no need to reset it for everything since not everything has the default ones